orion/controllers/programs/programsController.go

```diff
@@ -81,6 +84,42 @@ func DeleteProgram(c *gin.Context) {
   return
 }
 
-func CheckValidProgram() bool {
-  return true
+func CheckValidProgram(program Program) error {
+  // Retrieves the inputted values
+  programId := program.ProgramId
+  name := program.Name
+  grade1 := program.Grade1
+  grade2 := program.Grade2
+
+  // Checks if program name or program ID is empty
```

Comment:
You can remove the comments here. This part is checking if ProgramId is empty which is self-explanatory from the code